GroupProject/Pages/Index.cshtml.cs

```diff
@@ -35,24 +35,21 @@ public void OnGet()
 
 
 
-                string Membership = webClient.DownloadString("https://api.sportsdata.io/v3/soccer/scores/json/MembershipsByCompetition/EPL?key=bc49021bad1943008414c5a75e665961");
+                string playerInfoData = webClient.DownloadString("https://api.sportsdata.io/v3/soccer/scores/json/MembershipsByCompetition/EPL?key=bc49021bad1943008414c5a75e665961");
```

Comment:
For clarity of exactly what data is being read in, I changed "Membership" to "playerInfoData”

Suggestion to have clarity for this came from lectures

Reply:
Scrap the word "data".  It's a given that variables hold data.  If we put this on a variable, every variable would end with data.
Beyond that... Membership or Player Info...  I'm indifferent.